scripts/projector.md

```diff
@@ -0,0 +1,38 @@
+
+## Projector script documentation
+
+The projector installation and initialization script (see [`projector.sh`](projector.sh)) allows different levels of customization:
+
+### Projector config name
+
+Specify the **desired name of Projector config** by defining the `projector_config_name` variable.
+Otherwise it wil be "`Idea`".
+
+Example:
+
+```sh
+projector_config_name=Idea
+```
+
+### Projector IDE name
+
+Specify the **Projector IDE you want to install** by defining the `projector_ide_name` variable.
+Otherwise it wil be "`IntelliJ IDEA Community Edition 2020.3.4`" (latest projector-tested IntelliJ IDEA Community Edition).
+
+Note: to get list of available IDEs, launch `projector find`.
```

Comment:
Note: to get **a** list of available IDEs, launch `projector find`.

Reply:
Fixed into "get **the** list"